49.md

```diff
@@ -0,0 +1,100 @@
+NIP-49
+======
+
+Nostr Wallet Auth
+--------------------
+
+`draft` `optional`
+
+## Rationale
+
+[Nostr Wallet Connect](47.md) is a protocol for talking to a lightning wallet over nostr. It is great but hinges on
+having the user copy-paste a wallet connection URI into the app they wish to connect with. This can be a UX hurdle and
+often has the user handling sensitive information that they may not understand.
+
+As well, Nostr Wallet Connect has lots of different functions that a wallet may or may not support or an app may or may
+not need. This either runs into silent failures or requires the user to understand what functions their wallet supports
+and what functions the app needs.
+
+Finally, Nostr Wallet Connect requires the wallet to generate both key pairs and the app to use these. This limits the
+key management strategies that an app can use and often leads to suboptimal key management.
+
+This NIP proposes a new protocol that solves these problems by having the wallet and app generate a NWC connection URI
+together. This URI is then used to connect the wallet and app.
```

Comment:
This wording is a little confusing. They aren't making the exact same URI as NIP-47 since this proposal replaces symmetric encryption with public key.